packages/common/utils/src/object/omit.ts

```diff
@@ -1,12 +1,12 @@
 /** @tossdocs-ignore */
-import { ObjectKeys, objectKeys } from './object-keys';
+import { objectKeys } from './object-keys';
 import { ElementType } from './types';
 
-export function omit<ObjectType extends Record<PropertyKey, any>, KeyTypes extends Array<ObjectKeys<ObjectType>>>(
-  obj: ObjectType,
-  keys: KeyTypes
-) {
+export function omit<T extends Record<PropertyKey, T[keyof T]>, K extends Array<keyof T>>(
```

Comment:
1. The original code means "You should pass an object value to `obj`." so `unknown` seems appropriate. It doesn't have to be `T[keyof T]`.
2. The reason why `ObjectKeys` is used here is that `Object.keys` excludes symbol property, because symbol is not enumerable.

Reply:
@HoseungJang 
Oh, Thank you..! The second part is something I missed.

```ts
export type ObjectKeys<T extends Record<PropertyKey, unknown>> = `${Exclude<keyof T, symbol>}`;
```
One more question, is there any particular reason why you wrapped `Exclude` in backticks in the above sentence?
Do you think it would be okay to remove the backticks?

Reply:
The answer is no because `Object.keys` basically returns an array of strings. The backticks are the syntax of [template literal types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) and it is for converting `T` to string type.

Reply:
@HoseungJang 
I understand what you're saying. 

I initially thought it would be okay to remove the vectic because the type inference would be the same, but I understand your intent. 

Thanks.